Makefile

```diff
@@ -0,0 +1,5 @@
+BIN_DIR='bin/'
+
+.PHONY: build
+build:
+	mkdir -p ${BIN_DIR} && go build -o ${BIN_DIR} ./...
```

Comment:
Again, this is your personal configuration. Most contributors use 'go test ./...' or "go install ./cmd/starlark".

Reply:
Can we document this is the README? I added the minimal Makefile because there was the README talks about installing the CLI globally instead of building the repo locally.

Reply:
The install command you quoted is already document in the main README. We can add a note to say run `go test ./...`, but I think for most Go users that's the first thing they would try